dev/breeze/src/airflow_breeze/shell/enter_shell.py

```diff
@@ -117,18 +117,17 @@
 }
 
 
-def construct_arguments_docker_compose_command(shell_params: ShellBuilder) -> List[str]:
-    args_command = []
+def construct_env_variables_docker_compose_command(shell_params: ShellBuilder) -> Dict[str, str]:
```

Comment:
Rather than buildign a command - I simply return dictionary of variables that shoudl be used in the form of "key->value" dict. This will be easier to use when you run commands